### DISCUSSION TITLE
I can run Quarto in my RStudio terminal, but it's not in the UI (compose a Quarto Doc, ect)

### DISCUSSION THREAD

**Discussion by IceManTBM3** [quarto-dev/quarto-cli#7492] [Q&A]
### Description

I can run Quarto Check, Quarto Render and Preview in Terminal, but there is no Quarto Doc compose option. How do I get that to work? Thx in advance. 
![NoQuarto](https://github.com/quarto-dev/quarto-cli/assets/129208946/fff3c447-fa4f-4cb0-83a0-338510a89623)

1. Here is my check dump

$ quarto check
Quarto 1.4.451
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.8: OK
      Dart Sass version 1.55.0: OK
      Deno version 1.33.4: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.4.451
      Path: C:\Users\tmeek\AppData\Local\Programs\Quarto\bin
      CodePage: 1252

[>] Checking tools....................OK
      TinyTeX: v2023.05
      Chromium: 869685

[>] Checking LaTeX....................OK
      Using: TinyTex
      Path: C:\Users\tmeek\AppData\Roaming\TinyTeX\bin\windows\
      Version: 2023

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.9.7 (Conda)
      Path: C:/ProgramData/Anaconda3/python.exe
      Jupyter: 4.8.1
      Kernels: python3

[>] Checking Jupyter engine render....OK

[>] Checking R installation...........OK
      Version: 4.2.2
      Path: C:/Users/tmeek/AppData/Local/Programs/R/R-4.2.2
      LibPaths:
        - C:/Users/tmeek/AppData/Local/Programs/R/R-4.2.2/library
      knitr: 1.43
      rmarkdown: 2.25

[>] Checking Knitr engine render......OK

**Comment by mcanouil** [comment]
What is the version of RStudio you are using?

Did you check the getting started guide? <https://quarto.org/docs/get-started/hello/rstudio.html>

**Reply by IceManTBM3** [reply]
Version 1.3.959

**Reply by mcanouil** [reply]
It's been a while since RStudio adopted a version using year in it.
You should upgrade your RStudio as documented in the getting started guide.

**Reply by IceManTBM3** [reply]
Ok did that... now here's the deal, IF, I create a new Quarto Doc, I see the visual and source sub pane, but for existing .qmd files, I don't. hmmmm

**Reply by mcanouil** [reply]
I'm sorry but I don't understand what you are saying.

**Reply by IceManTBM3** [reply]
It's Ok... the docs needed to be closed and reOpened.

Thx for your help!

**Comment by IceManTBM3** [comment]
Will do, thx
